q2_sample_classifier/visuals.py

```diff
@@ -149,6 +153,27 @@ def _plot_confusion_matrix(y_test, y_pred, classes, normalize, palette):
     return predictions, confusion
 
 
+def _check_vmin_and_vmax(cm, vmin, vmax):
+    lowest_frequency = np.amin(cm)
+    highest_frequency = np.amax(cm)
+
+    error = ''
+    if vmin is not None:
+        if vmin > lowest_frequency:
+            error += ('Your vmin value must be less than or equal to the '
+                      'lowest prediction frequency present:\n'
+                      f'\t{vmin!r} is greater than {lowest_frequency!r}')
+    if vmax is not None:
+        if vmax < highest_frequency:
+            if error:
+                error += '\n'
+            error += ('Your vmax value must be greater than or equal to the '
```

Comment:
Please change to:
"vmax must be greater than or equal to the highest predicted class frequency"